StarRating/themes/StarRating_template.less

```diff
@@ -3,8 +3,9 @@
 .d-star-rating{
 	/* Do not modify the display style */
 	display: inline-block;
-	font-size: @star-icon-size;
+	font-size: 285%;
```

Comment:
Fixed in https://github.com/dmandrioli/deliteful/commit/9178ec14b6f60258da8b85dae726537240c04767